controllers/JobSeekerController.js

```diff
@@ -0,0 +1,116 @@
+const { body, validationResult, sanitizeBody } = require("express-validator");
+const apiResponse = require("../helpers/apiResponse");
+const { constants } = require("../helpers/constants");
+const auth = require("../middlewares/jwt");
+const UserModel = require("../models/UserModel");
+const JobSeekerModel = require("../models/JobSeekerModel");
+
+exports.JobSeeker = [
+  auth,
+  body("experienceType", "Experience Type must not be empty.").notEmpty(),
+  body("experience", "Experience must not be empty.").isNumeric(),
+  body("currentRole", "current Role must not be empty.").notEmpty(),
+  body("skills", "skills must not be empty.").notEmpty().isArray({ max: 4 }),
+  body("currentLocation", "current Location must not be empty.").notEmpty(),
+  body("openToWork", "openTo Work must not be empty.").notEmpty().isArray(),
+  body("currentCTC", "current CTC must not be empty.").isNumeric(),
+  body("noticePeriod", "notice Period must not be empty.").isNumeric(),
+  body("resume", "resume must not be empty.").notEmpty(),
+  body("about", "about must not be empty.").notEmpty().isLength({ max: 500 }),
+  (req, res) => {
+    try {
+      const user = req.user;
+
+      if (user.userType !== constants.userTypes.JOBSEEKER)
+        return apiResponse.unauthorizedResponse(res, "Invalid Permission.");
+
+      const errors = validationResult(req);
+      if (!errors.isEmpty()) {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 3 tabs but found 6 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
			if (!errors.isEmpty()) {
```